README.md

```diff
@@ -78,16 +78,16 @@ You can configure the logging agent to filter log lines according to the Kuberne
 For example, to set the agent to return all log lines coming from the `default` Kubernetes namespace and exclude anything with a label `app.kubernetes.io/name` with value `sample-app` or an annotation `annotation.user` with value `sample-user`, include these parameters:
 
 ```text
-custom_logdna_at_agent_line_exclusion = "label.app.kubernetes.io/name:sample-app\\, annotation.user:sample-user"
-custom_logdna_at_agent_line_inclusion = "namespace:default"
+custom_log_analysis_at_agent_line_exclusion = "label.app.kubernetes.io/name:sample-app\\, annotation.user:sample-user"
+custom_log_analysis_at_agent_line_inclusion = "namespace:default"
 ```
 
 The following is the corresponding DaemonSet configuration:
 
 ```text
-- name: LOGDNA_K8S_METADATA_LINE_INCLUSION
+- name: LOG_ANALYSIS_K8S_METADATA_LINE_INCLUSION
   value: "label.app.kubernetes.io/name:sample-app, annotation.user:sample-user"
-- name: LOGDNA_K8S_METADATA_LINE_EXCLUSION
+- name: LOG_ANALYSIS_K8S_METADATA_LINE_EXCLUSION
```

Comment:
This actually should be `LOGDNA_K8S_METADATA_LINE_EXCLUSION` - please change it back